package.json

```diff
@@ -30,9 +30,9 @@
   },
   "homepage": "http://developers.smartsheet.com",
   "dependencies": {
+    "axios": "^1.4.0",
```

Comment:
[Apparently](https://github.com/isaacs/github/issues/1655) github doesn't allow comments on lines that weren't changed in the PR 😵 

Anyway, can you also bump the version of the SDK package? (Line 3)

Reply:
On top of this could you also update the CHANGELOG.md please

Reply:
Yes I can do that! I will bump a minor version since these changes aren't breaking to the contract but are fairly wide reaching.

Reply:
Done in latest commit. Bumped to v3.2.0.

Reply:
Makes sense to me! Updated version to 3.1.3 in latest commit